CHANGELOG.md

```diff
@@ -291,17 +320,34 @@
 - `[store]` When pruning force compaction of the database. ([\#1972](https://github.com/cometbft/cometbft/pull/1972))
 - `[metrics]` Added metrics to monitor state store access. ([\#1974](https://github.com/cometbft/cometbft/pull/1974))
 - `[metrics]` Added metrics to monitor block store access. ([\#1974](https://github.com/cometbft/cometbft/pull/1974))
+`[config]` Removed unused `[mempool.max_batch_bytes]` mempool parameter
+ ([\#2056](https://github.com/cometbft/cometbft/pull/2056/))
```

Comment:
need to fix format in original `.changelog` file

Reply:
fixed